intermediate-css-picasso-painting/index-final.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+
+    <html lang="en" dir="auto">
+
+        <head>
+            <meta charset="utf-8">
+
+            <title>freeCodeCamp Curriculum - Learn intermediate CSS by building a Picasso painting</title>
+
+            <!-- Link to Fontawesome icons -->
+            <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css" integrity="sha384-oS3vJWv+0UjzBfQzYUhtDYW+Pj2yciDJxpsK1OYPAYjqT085Qq/1cq5FLXAZQ7Ay" crossorigin="anonymous">
+
+            <!-- Link to stylesheet -->
+            <link href="css/style-final.css" rel="stylesheet" type="text/css" media="screen">
+        </head>
+
+        <body>
+
+            <!--
```

Comment:
Since this is the final version of `index.html` the comments here can also be removed. And to make things easier for learners, and for you writing lessons, we could shorten the title to something like "CSS Three Musicians". We could also do things like include the Font Awesome icons locally, and remove a lot of the attributes like `dir` and `media`, along with removing the `meta` element entirely.

Sorry, I should have caught this all earlier. Originally I thought we would include some of the HTML code for the learners, but we're moving in the direction of having them write as much as possible, especially for these early projects.

Reply:
The final version of this file could look something like this:

```
<!DOCTYPE html>
<html lang="en">
<head>
  <title>CSS Three Musicians</title>
  <link rel="stylesheet" href="css/font-awesome/all.css">
  <link href="css/style-final.css" rel="stylesheet" type="text/css">
</head>

<body>
  <div id="back-wall"></div>
  <div class="characters">
    <div id="offwhite-character">
      <div id="white-hat"></div>
      <div id="black-mask">
        <div class="eyes left"></div>
        <div class="eyes right"></div>
      </div>
      <div id="gray-instrument">
        <div class="black-dot"></div>
        <div class="black-dot"></div>
        <div class="black-dot"></div>
        <div class="black-dot"></div>
        <div class="black-dot"></div>
      </div>
      <div id="tan-table"></div>
    </div>
    <div id="black-character">
      <div id="black-hat"></div>
      <div id="gray-mask">
        <div class="eyes left"></div>
        <div class="eyes right"></div>
      </div>
      <div id="white-paper">
        <i class="fas fa-music"></i>
        <i class="fas fa-music"></i>
        <i class="fas fa-music"></i>
        <i class="fas fa-music"></i>
      </div>
    </div>
    <div class="blue" id="blue-left"></div>
    <div class="blue" id="blue-right"></div>
    <div id="orange-character">
      <div id="black-round-hat"></div>
      <div id="eyes-div">
        <div class="eyes left"></div>
        <div class="eyes right"></div>
      </div>
      <div id="triangles">
        <div class="triangle"></div>
        <div class="triangle"></div>
        <div class="triangle"></div>
        <div class="triangle"></div>
        <div class="triangle"></div>
        <div class="triangle"></div>
        <div class="triangle"></div>
        <div class="triangle"></div>
        <div class="triangle"></div>
        <div class="triangle"></div>
        <div class="triangle"></div>
        <div class="triangle"></div>
        <div class="triangle"></div>
        <div class="triangle"></div>
        <div class="triangle"></div>
        <div class="triangle"></div>
        <div class="triangle"></div>
        <div class="triangle"></div>
        <div class="triangle"></div>
        <div class="triangle"></div>
        <div class="triangle"></div>
        <div class="triangle"></div>
        <div class="triangle"></div>
        <div class="triangle"></div>
        <div class="triangle"></div>
        <div class="triangle"></div>
        <div class="triangle"></div>
        <div class="triangle"></div>
        <div class="triangle"></div>
        <div class="triangle"></div>
      </div>
      <div id="guitar">
        <div class="guitar" id="guitar-left">
          <i class="fas fa-bars"></i>
        </div>
        <div class="guitar" id="guitar-right">
          <i class="fas fa-bars"></i>
        </div>
        <div id="guitar-neck"></div>
      </div>
    </div>
  </div>
</body>
```